sass/patternfly-react/_notificationdrawer.scss

```diff
@@ -0,0 +1,45 @@
+.drawer-pf.drawer-alt-pf {
+  overflow-y: hidden;
+  display: flex;
+  flex-direction: column;
+
+  .drawer-pf-title {
+    position: relative;
+  }
+
+  .panel-group {
+    display: flex;
+    flex-direction: column;
+    position: initial;
+    bottom: initial;
+    top: initial;
+    overflow-y: auto;
+
+    .panel.panel-default.expanded {
+      min-height: 175px;
+      flex: 1 1 auto;
+      display: flex;
+      flex-direction: column;
+    }
+
+    .panel-collapse.in {
+      display: flex;
+      flex-direction: column;
+      flex: 1 1 auto;
+
```

Comment:
If you add `min-height: 0;`, this will fix a scrolling issue that is occurring in Firefox.